virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc

```diff
@@ -1,71 +1,44 @@
 :_content-type: ASSEMBLY
-[id="attaching-to-multiple-networks"]
-= Attaching a virtual machine to multiple networks
+[id="virt-attaching-vm-multiple-networks"]
+= Attaching a virtual machine to a Linux bridge network
 include::_attributes/common-attributes.adoc[]
-:context: virt-attaching-multiple-networks
+:context: virt-attaching-vm-multiple-networks
 
 toc::[]
 
-You can attach virtual machines to multiple networks by using Linux bridges.
-You can also import virtual machines with existing workloads that depend on access to multiple interfaces.
+By default, {VirtProductName} is installed with a single, internal pod network.
 
-To attach a virtual machine to an additional network:
-
-. Configure a Linux bridge.
-. Configure a bridge network attachment definition for a namespace in the web console or CLI.
-+
-[NOTE]
-====
-The network attachment definition must be in the same namespace as the pod or virtual machine.
-====
-. Attach the virtual machine to the network attachment definition by using either the web console or the CLI:
-** In the web console, create a NIC for a new or existing virtual machine.
-** In the CLI, include the network information in the virtual machine configuration.
-
-[NOTE]
-====
-There are multiple methods for configuring a VLAN, including network attachment definition and node network configuration policy.
-However, a network attachment definition provides a more efficient and more manageable configuration.
-====
+You must create a Linux bridge network attachment definition (NAD) in order to connect to additional networks.
 
-include::modules/virt-networking-glossary.adoc[leveloffset=+1]
+= Connecting to the network through the network attachment definition
 
-[id="virt-creating-linux-bridge"]
-== Configuring a Linux bridge
+To attach a virtual machine to an additional network:
 
-include::modules/virt-creating-linux-bridge-nncp.adoc[leveloffset=+2]
+. Create a Linux bridge node network configuration policy.
+. Create a Linux bridge network attachment definition.
+. Configure the virtual machine, enabling the virtual machine to recognize the NAD.
```

Comment:
Very small "nit": Although we use "NAD" informally as an abbreviation, in the docs it only occurs in the API sections. I think it would be a good idea to spell it in full ("network attachment definition"), for consistency. That is how it is referred to in the Network sections.

Reply:
Done.